pkg/process/config/config.go

```diff
@@ -383,7 +385,7 @@ func initRuntimeSettings() {
 
 	// Before we begin listening, register runtime settings
 	for _, setting := range processRuntimeSettings {
-		_ = log.Warn(settings.RegisterRuntimeSetting(setting))
+		_ = settings.RegisterRuntimeSetting(setting)
```

Comment:
Looking at other usages of `RegisterRuntimeSettings`, we should probably check for errors and return it in the `initRuntimeSettings` function. 

The function that is calling `initRuntimeSettings` can then probably log an WARN message to inform the user that runtime settings weren't successfully loaded.

Taking `system-probe` as an example:
https://github.com/DataDog/datadog-agent/blob/fbe9b1e405b43baebafa01892628dd46c90aaf76/cmd/system-probe/app/config.go#L155-L158

https://github.com/DataDog/datadog-agent/blob/a8fb64c24cf0cf83c207a3fa3bd6e9e8b0e9a46f/cmd/system-probe/app/run.go#L150-L152

Reply:
From what I could tell the only time RegisterRuntimeSetting returns is if there is a duplicate. I figured it wasn't really a big enough error to output, and ignoring it would allow you to run this twice. However you bring up a good point that we should mimic the behavior of other agents, so I will edit this.